egs/swbd/s5c/local/nnet3/run_reduced_frame.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This scripts does nnet3-tdnn training and decoding 
```

Comment:
As regards the naming of the scripts, we need to use a convention that will scale as we do more
experiments.  Let's use 'lfr' (for low frame-rate) as the extension, and you can create the file
`tuning/run_tdnn_lfr1a.sh`
and make a soft link
run_tdnn_lfr.sh -> tuning/run_tdnn_lfr1a.sh
Remember to document in a comment, which specific script in tuning/ it was copied from,
and if possible add in a comment the summary line generated from steps/info/nnet3_dir_info.pl (excessive line length is OK), and
from local/nnet3/compare_wer_general.sh or whatever we now call that script, to compare
it with the baseline... if you don't have the baseline locally, just compare it with itself and
manually copy the numbers from the baseline to fake it.